constants/nav_links.js

```diff
@@ -0,0 +1,22 @@
+export const NAV_LINKS = [
```

Comment:
We may or may. not need this. Please remove for now or just keep 1 page.

Reply:
I removed it